flexmeasures/api/v3_0/sensors.py

```diff
@@ -445,11 +445,22 @@ def get_schedule(  # noqa: C901
 
         # Look up the scheduling job
         connection = current_app.queues["scheduling"].connection
+
         try:  # First try the scheduling queue
             job = Job.fetch(job_id, connection=connection)
         except NoSuchJobError:
             return unrecognized_event(job_id, "job")
 
+        if (
+            not current_app.config.get("FLEXMEASURES_FALLBACK_REDIRECT")
+            and job.is_failed
+            and ("fallback_job_id" in job.meta)
+        ):
+            try:  # First try the scheduling queue
+                job = Job.fetch(job.meta["fallback_job_id"], connection=connection)
+            except NoSuchJobError:
+                return unrecognized_event(job.meta["fallback_job_id"], "fallback-job")
```

Comment:
Does this case already log an informative server error? We don't expect it to ever happen that a job states a non-existing fallback job ID, right? I think the server error log should show the fallback job ID, but also the original job ID. But perhaps it does already.

Reply:
Sure, let's log both. This is unlikely to happen but possible. I guess that this could happen if the job gets deleted (manually) or while we clean up the queue (for example, a re-deployment event).